frontend/src/components/core/Sidebar/SidebarNav.test.tsx

```diff
@@ -215,129 +201,148 @@ describe("SidebarNav", () => {
+      <SidebarNav
+        {...getProps({
+          hasSidebarElements: true,
+          appPages: [
+            { pageName: "my_first_page" },
+            { pageName: "my_second_page" },
+            { pageName: "my_third_page" },
+            { pageName: "my_fourth_page" },
+            { pageName: "my_fifth_page" },
+            { pageName: "my_sixth_page" },
+            { pageName: "my_seventh_page" },
+            { pageName: "my_eight_page" },
+          ],
+        })}
+      />
     )
 
-    expect(
-      wrapper.find(StyledSidebarNavSeparatorContainer).find(Icon).props()
-    ).toHaveProperty("content", ExpandMore)
+    expect(wrapper.find(StyledSidebarNavButton).text()).toBe("2 More")
   })
 
-  it("renders ExpandLess icon when expanded and not overflowing", () => {
+  it("renders the correct StyledSidebarNavButton when expanded", () => {
     // We need to have useIsOverflowing return true once so that we can click
     // on the separator to expand the nav component. After this click, it
     // returns false.
     mockUseIsOverflowing.mockReturnValueOnce(true)
     const wrapper = shallow(
-      <SidebarNav {...getProps({ hasSidebarElements: true })} />
-    )
-
-    wrapper.find(StyledSidebarNavSeparatorContainer).prop("onClick")!(
-      mockClickEvent
-    )
-    expect(
-      wrapper.find(StyledSidebarNavSeparatorContainer).find(Icon).props()
-    ).toHaveProperty("content", ExpandLess)
-  })
-
-  it("renders ExpandLess icon when expanded and overflowing", () => {
-    // Have useIsOverflowing return true both before and after the nav is
-    // expanded.
-    mockUseIsOverflowing.mockReturnValueOnce(true).mockReturnValueOnce(true)
-    const wrapper = shallow(
-      <SidebarNav {...getProps({ hasSidebarElements: true })} />
+      <SidebarNav
+        {...getProps({
+          hasSidebarElements: true,
+          appPages: [
+            { pageName: "my_first_page" },
+            { pageName: "my_second_page" },
+            { pageName: "my_third_page" },
+            { pageName: "my_fourth_page" },
+            { pageName: "my_fifth_page" },
+            { pageName: "my_sixth_page" },
+            { pageName: "my_seventh_page" },
+            { pageName: "my_eight_page" },
+          ],
+        })}
+      />
     )
 
-    wrapper.find(StyledSidebarNavSeparatorContainer).prop("onClick")!(
-      mockClickEvent
-    )
-    expect(
-      wrapper.find(StyledSidebarNavSeparatorContainer).find(Icon).props()
-    ).toHaveProperty("content", ExpandLess)
+    wrapper.find(StyledSidebarNavButton).prop("onClick")!(mockClickEvent)
+    expect(wrapper.find(StyledSidebarNavButton).text()).toBe("View less")
   })
 
-  it("changes cursor to pointer above separator when overflowing", () => {
-    mockUseIsOverflowing.mockReturnValueOnce(true)
-    // Need mount > shallow here so that toHaveStyleRule can be used.
+  it("is collapsed by default if there are widgets on the sidebar", () => {
```

Comment:
Since this condition is no longer the case (collapse/expanded default passed via `collapseNav` prop) - it would be great to convert this to confirm that `hasSidebarElements: true` renders the `StyledSidebarNavSeparatorContainer` instead, along with the related reverse scenario test, `hasSidebarElements: false` results in no `StyledSidebarNavSeparatorContainer` .